pyproject.toml

```diff
@@ -34,7 +34,7 @@ classifiers = [
 cdflib = "==1.2.6"
 imap-data-access = ">=0.5.0"
 python = ">=3.9,<4"
-space_packet_parser = ">=4.2.0"
+space_packet_parser = "^4.2.0"
```

Comment:
The release of `space_packet_parser 5.0.0` has breaking changes so for now I pinned the version less than 5.